SeaPublicWebsite.UnitTests/QuestionFlowServiceTests.cs

```diff
@@ -78,18 +78,6 @@ public void RunSkipLinkTestCases(QuestionFlowServiceTestCase testCase)
                 nameof(EnergyEfficiencyController.NewOrReturningUser_Get),
                 "EnergyEfficiency"
             )),
-        new(
-            "Changing country goes to Summary",
-            new Input(
-                QuestionFlowPage.Country,
-                "ABCDEFGH",
-                entryPoint: QuestionFlowPage.Country
-            ),
-            new PathByActionArguments(
-                nameof(EnergyEfficiencyController.AnswerSummary_Get),
-                "EnergyEfficiency",
-                new { reference = "ABCDEFGH" }
-            )),
```

Comment:
Removed change tests and actions for propery / house / bungalow / flat type, country and ownership status.

(You can still change your property type from answer summary, but you'd need to go through the whole flow. We'll fix this in one of the newer tickets detailing the new flow)